transcom/mymove
# B-20532 TOO queue assignment

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20532)
## [INT PR](https://github.com/transcom/mymove/pull/13835)

## Summary

Part of the effort towards QUEUE MANAGEMENT! This work allows TOO users/supervisors the basic functionality of QUEUE MANAGEMENT - assign, unassign, and reassign.

Because SC and TOO queues share ListOrders, it was updated to accept a role parameter. ListOrders will return and filter different results for the 'Assigned' column depending on the roletype argument it receives.

### How to test

WITH FF ON
1. Login as a TOO supervisor with multiple TOOs working at their transportation office
2. View your queue and note the `Assigned` column
3. Play around and assign different moves to different users
4. Refresh make sure the value is retained
5. Sort and filter and make sure it works (note: after the third sort, the assigned sort breaks - we believe this is an issue with the table itself, and have created an issue for it to be worked)
6. Try and break it (but don't try TOO hard)
